libavfilter/signature_lookup.c

```diff
@@ -199,7 +199,7 @@ static MatchingInfo* get_matching_parameters(AVFilterContext *ctx, SignatureCont
 
     /* initialize houghspace */
     for (i = 0; i < MAX_FRAMERATE; i++) {
-        hspace[i] = av_malloc_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem));
+        hspace[i] = av_mallocz_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem)); 
```

Comment:
```suggestion
        hspace[i] = av_mallocz_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem));
```

Oops, found a trailing whitespace from my last suggestion. Feel free to add this before squash.